src/transforms/branch.ts

```diff
@@ -1,29 +1,24 @@
 import { getJiraId } from "../jira/util/id";
 import issueKeyParser from "jira-issue-key-parser";
 import { getJiraAuthor } from "../util/jira";
-import _ from "lodash";
+import { isEmpty } from "lodash";
 import { WebhookPayloadCreate } from "@octokit/webhooks";
 import { GitHubAPI } from "probot";
 import { generateCreatePullRequestUrl } from "./util/pullRequestLinkGenerator";
+import GitHubClient from "../github/client/github-client";
+import { getCloudInstallationId } from "../github/client/installation-id";
 import { booleanFlag, BooleanFlags } from "../config/feature-flags";
+import { LoggerWithTarget } from "probot/lib/wrap-logger";
+import { JiraBranchData, JiraCommit } from "src/interfaces/jira";
 
-async function getLastCommit(github: GitHubAPI, webhookPayload: WebhookPayloadCreate, issueKeys: string[]) {
+const getLastCommit = async (useNewGitHubClient: boolean, github: GitHubAPI, gitHubClient: GitHubClient, webhookPayload: WebhookPayloadCreate, issueKeys: string[]): Promise<JiraCommit> => {
```

Comment:
Don't add in the old github client, and the new one, and the flag of which one to use as parameters of the function.

Instead you can just use a single client of type `GithubAPI | GithubClient` and then do an `instanceof` check to see which function to call.  Check my last PR as an example of this: https://github.com/atlassian/github-for-jira/pull/1113/files#diff-90aa83dd1c1806edf7ae34e6163d76150bf0e0e132556c007f8dabd0d9fbb2e7R21

It'll make the transition to removing octokit much simpler and the functions cleaner